chimera.mk

```diff
@@ -75,6 +75,19 @@ regenerate_soc_regs: $(CHIM_ROOT)/hw/regs/chimera_reg_pkg.sv $(CHIM_ROOT)/hw/reg
 $(CHIM_ROOT)/hw/regs/chimera_reg_pkg.sv $(CHIM_ROOT)/hw/regs/chimera_reg_top.sv: $(CHIM_ROOT)/hw/regs/chimera_regs.hjson
 	python $(CHIM_ROOT)/utils/reggen/regtool.py -r $< --outdir $(dir $@)
 
+
+# Nonfree components
+CHIM_NONFREE_REMOTE ?= git@iis-git.ee.ethz.ch:pulp-restricted/chimera-nonfree.git
+CHIM_NONFREE_COMMIT ?= 48fafe39
```

Comment:
For the future, one option to avoid excessive updates of this commit hash when updating the internal CI is to use a tag that gets moved in the internal repository (see iDMA), but I think this is fine for now.

Reply:
Also did this, since I was at it.